contents/handbook/company/diversity.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Diversity
+sidebar: Handbook
+showTitle: true
+---
+
+PostHog is proud to be an international group of misfits. You can't disrupt an industry by thinking the same way as everyone else.
+
+## How diversity helps us
+
+Having a great breadth of talented team members who think differently because of their different life stories, backgrounds, countries and levels of experience helps us better understand how to make sure there are more succesful products in the *world*.
+
+There are 10s of millions of people working in software development globally, and greater diversity means a greater understanding of our users.
+
+## Diversity as a tactic
+
+We view diversity as a critical tactic to achieve great performance, but not as an objective in its own right. Why?
+
+If we were to aim for diversity whilst also aiming for performance, then that would imply diversity is an ethical mission. We view this as a wonderful added benefit but it's not the *fundamental reason* for encouraging diversity at PostHog. Underrepresented great talent is *particularly great talent*, that *any* business should want.
```

Comment:
"We view this" -> what? It's confusing in relation to the previous sentence

Reply:
I don't quite understand this entire section (possibly because of this sentence). Could be worth elaborating a little bit more here

Reply:
this is covered in point above